lua/config/init.lua

```diff
@@ -0,0 +1,35 @@
+local M = {
+  USER_CONF_PATH = "~/.config/lvim/config.lua",
```

Comment:
We were still supporting lv-config.lua.  This makes it so that filename is no longer supported.  I think we should keep supporting it for a bit longer.